radis/spectrum/spectrum.py

```diff
@@ -2416,12 +2416,33 @@ def plot(
         plot_by_parts=False,
         show=True,
         show_ruler=False,
+        plotting_library="auto",
```

Comment:
```suggestion
        plotting_library="default",
```

And add comments about the available values in the plot function docstrings

Reply:
commited suggestion. working on docstrings for all changes i made, will release them asap